lib/csdl2openapi.js

```diff
@@ -1325,6 +1325,10 @@ module.exports.csdl2openapi = function (
                             type: 'object',
                             title: 'Collection of ' + nameParts(type.$Type || 'Edm.String').name,
                             properties: {
+                                [csdl.$Version > '4.0' ? '@count' : '@odata.count']: { 
+                                    type: 'number',
+                                    description: `The count name/value pair represents the number of entities in the collection. Available when using the [$count](${csdl.$Version > '4.0' ? 'http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html#sec_SystemQueryOptioncount' : 'http://docs.oasis-open.org/odata/odata/v4.0/odata-v4.0-part1-protocol.html#_The_$inlinecount_System'}) query option.`,

```

Comment:
```suggestion
                                    description: 'The number of entities in the collection. Available when using the [$count](http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html#sec_SystemQueryOptioncount) query option.',
```

No need to differentiate here between 4.0 and 4.01, the other system query options also reference the latest version 4.01.